src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -92,6 +56,21 @@ const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) =>
 
     const rows = table.getRowModel().rows.slice(0, TABLE_VISIBLE_ROWS)
 
+    if (error) {
+        return (
+            <Flex.Box justify="center" mt="5x">
+                <Typography.Paragraph
+                    align="left"
+                    padding_block="3x"
+                    padding_inline="6x"
+                    font_family="UBUNTU"
+                    textcolor="brand"
+                >
+                    There was an error fetching the live pricing data
```

Comment:
Should we translate error message? And probably make it red

Reply:
Thanks @NikitK-deriv , it's better to have this translated. textcolor="brand" is making it red.